### PR TITLE
Improving documentation + Adding Manifest

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related to: https://github.com/demisto/etc/issues/39326

## Description
Added app manifest and improved documentation.

### CODE REVIEW

**Comment by amshamah419** [comment]
Mainly note the documentation here - https://github.com/demisto/content/blob/slack-docs-manifest/Packs/Slack/Integrations/SlackV3/README.md

I'm working through the steps with a customer currently and updating where we run into difficulties.

File: Packs/Slack/Integrations/SlackV3/SlackV3.py [hunk 1]
```diff
@@ -1976,14 +1968,11 @@ def main() -> None:
     commands = {
         'test-module': test_module,
         'long-running-execution': long_running_main,
-        'slack-mirror-investigation': mirror_investigation,
```
Comment (amshamah419) [inline Packs/Slack/Integrations/SlackV3/SlackV3.py:1979]: These were deprecated in V2, safe to remove.

File: Packs/Slack/Integrations/SlackV3/SlackV3.yml [hunk 2]
```diff
@@ -80,10 +80,6 @@ configuration:
   name: paginated_count
   required: false
   type: 0
-- display: Proxy URL to use in Slack API calls
```
Comment (amshamah419) [inline Packs/Slack/Integrations/SlackV3/SlackV3.yml:83]: Changed how we handle the proxies to be more inline with current standards

File: Packs/Slack/Integrations/SlackV3/SlackV3_description.md [hunk 3]
```diff
@@ -1,61 +1,4 @@
-## Configuring Slack V3
```
Comment (amshamah419) [inline Packs/Slack/Integrations/SlackV3/SlackV3_description.md:1]: Dedupped the setup instructions to minimize duplicate docs

**Review by yuvalbenshalom — APPROVED** [review]

**Review by glicht — APPROVED** [review]

**Pull request merged**